### PR TITLE
#163384175 Add all the required env vars in the image

### PR DESCRIPTION
**Pull request by mirr254**
#### What does this PR do?
Adds new environment variables in terraform scripts and in VOF image creation.
#### Description of Task to be completed?
- update `terraform/vof/compute.tf` with the required variables
- create new terraform variables `terraform/vof/variables.tf` to hold the keys
- update the start script to export the newly environment variables

#### How should this be manually tested?
- `clone` this repo
- `git checkout ft-include-new-login-keys-163384175`
- deploy this to local folk or sandbox
- `SSH` into the sandbox via `jumpbox` to check if they have been set

#### Any background context you want to provide?
There are keys that are required if user login is supposed to work properly. These keys include
```
ANDELA_MICRO_PUBLIC_KEY
LEARNER_MICRO_PUBLIC_KEY
```
#### What are the relevant pivotal tracker stories?
[163384175](https://www.pivotaltracker.com/story/show/163384175)
#### Questions:

### CODE REVIEW

File: packer/start_vof.sh [hunk 1]
```diff
@@ -22,6 +22,8 @@ export RAILS_ENV="$(get_var "railsEnv")"
 export REDIS_IP=$(get_var "redisIp")
 export BUGSNAG_KEY="$(get_var "bugsnagKey")"
 export DEPLOY_ENV="$(get_var "railsEnv")"
+export ANDELA_MICRO_PUBLIC_KEY="$(get_var "andela_micro_public_key")"
+export LEARNER_MICRO_PUBLIC_KEY="$(get_var "LEARNER_MICRO_PUBLIC_KEY")"
```
Comment (bigzoo) [inline packer/start_vof.sh]:
```suggestion
export LEARNER_MICRO_PUBLIC_KEY="$(get_var "learner_micro_public_key")"
```

File: vof/compute.tf [hunk 2]
```diff
@@ -51,26 +51,28 @@ resource "google_compute_instance_template" "vof-app-server-template" {
...
+    slackWebhook                           = "${var.slack_webhook_url}"
+    startup-script                         = "/home/vof/start_vof.sh"
+    serial-port-enable                     = 1
+    userMicroserviceApiUrl                 = "${var.user_microservice_api_url}"
+    userMicroserviceApiToken               = "${var.user_microservice_api_token}"
+    google_storage_access_key_id           = "${var.google_storage_access_key_id}"
+    google_storage_secret_access_key       = "${var.google_storage_secret_access_key}"
+    dbBackupNotificationToken              = "${var.db_backup_notification_token}"
+    andelaMicroPublicKey                = "${var.andela_micro_public_key}"
+    learnerMicroPublicKey               = "${var.learner_micro_public_key}"
```
Comment (anyric) [inline vof/compute.tf]: align the assignments to be consistent

File: vof/compute.tf [hunk 3]
```diff
@@ -71,10 +71,8 @@ resource "google_compute_instance_template" "vof-app-server-template" {
     google_storage_access_key_id           = "${var.google_storage_access_key_id}"
```
Comment (Thegaijin) [inline vof/compute.tf]: All these variables should be camel case, please correct it

Comment (mirr254) [reply vof/compute.tf]: I will look into that but it wasn't part of the PR, they have been there all along.

**Pull request merged**